ocs_ci/ocs/cluster.py

```diff
@@ -514,31 +514,46 @@ def get_ceph_status(self, format=None):
             cmd += f" -f {format}"
         return self.toolbox.exec_cmd_on_pod(cmd, out_yaml_format=False)
 
+    def get_ceph_default_replica(self):
+        """
+        The function return the default replica count in the system,
+        taken from 'ceph status'. in case no parameter found, return '0'.
+
+        Returns:
+             int : the default replica count - 0 if not found.
+        """
+        ceph_pod = pod.get_ceph_tools_pod()
+        ceph_status = ceph_pod.exec_ceph_cmd(ceph_cmd="ceph status")
+        av_mod = ceph_status.get("mgrmap").get("available_modules")
+        for mod in av_mod:
+            if mod["name"] == "localpool":
+                return mod.get("module_options").get("num_rep").get("default_value")
+        logger.error("Replica count number did not found !")
+        return 0
```

Comment:
isn't the replica  '1'  if there is no replica set.

Reply:
we have no deployment with replica-1, so i do not know what this code will return on system with replica-1
this `return 0` is in case there is a problem and the num_rep parameter is not define.